<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Key:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="keyLineEdit">
         <property name="placeholderText">
          <string>Key</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchButton">
         <property name="toolTip">
          <string>Toggle search mode.</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resource_files/resources.qrc">
           <normaloff>:/icons/search.png</normaloff>
           <activeon>:/icons/search_highlighted.png</activeon>:/icons/search.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteEntryButton">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resource_files/resources.qrc">
           <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Value:</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QFrame" name="valueFrame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QComboBox" name="valueTypeComboBox"/>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QPushButton" name="selectValueFileInExplorerButton">
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Select</string>
         </property>
         <property name="icon">
          <iconset resource="../resource_files/resources.qrc">
           <normaloff>:/icons/select.png</normaloff>:/icons/select.png</iconset>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QPushButton" name="chooseFileButton">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resource_files/resources.qrc">
           <normaloff>:/icons/file.png</normaloff>:/icons/file.png</iconset>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QPushButton" name="chooseDirButton">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resource_files/resources.qrc">
           <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QPushButton" name="addButton">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Add/Modify</string>
         </property>
         <property name="icon">
          <iconset resource="../resource_files/resources.qrc">
           <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="settingsTableView">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>keyLineEdit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>deleteEntryButton</tabstop>
  <tabstop>selectValueFileInExplorerButton</tabstop>
  <tabstop>chooseFileButton</tabstop>
  <tabstop>chooseDirButton</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>settingsTableView</tabstop>
 </tabstops>
 <resources>
  <include location="../resource_files/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>keyLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
